superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeControl.test.tsx

```diff
@@ -31,8 +31,8 @@ import { testWithId } from 'src/utils/testUtils';
 import {
   EchartsMixedTimeseriesChartPlugin,
   EchartsTimeseriesChartPlugin,
-} from '@superset-ui/plugin-chart-echarts/lib';
-import { LineChartPlugin } from '@superset-ui/preset-chart-xy/lib';
+} from '@superset-ui/plugin-chart-echarts';
+import { LineChartPlugin } from '@superset-ui/preset-chart-xy';
 import TimeTableChartPlugin from '../../../../visualizations/TimeTable/TimeTableChartPlugin';
```

Comment:
We should probably move this viz plugin to its own package as part of the bigger monorepo effort